chat-sdk/src/main/java/com/amazon/connect/chat/sdk/repository/ChatService.kt

```diff
@@ -571,6 +610,7 @@ class ChatServiceImpl @Inject constructor(
                 attachmentId = UUID.randomUUID().toString(),  // Temporary attachmentId
                 displayName = getRecentDisplayName()
             )
+            tempMessageIdToFileUrl[recentlySentAttachmentMessage!!.id] = fileUri
```

Comment:
In general, we do want to avoid using `!!` since it'll throw a NPE and stop execution if it fails.  Here is a workaround:

```
recentlySentAttachmentMessage?.let { message ->
    tempMessageIdToFileUrl[message.id] = fileUri
    sendSingleUpdateToClient(message)
}
```

The code block will only execute if `recentlySentAttachmentMessage` is defined.

Reply:
Makes sense, will update